oeq-ts-rest-api/src/UserQuery.ts

```diff
@@ -61,6 +61,15 @@ export interface SearchParams {
   users: boolean;
 }
 
+export interface LookupParams {
+  /** String array of User IDs to lookup. */
```

Comment:
No need to put the type info in the comment. So this (and the below three) can be:

```suggestion
  /** User IDs to lookup. */
```

Or if you want more, keep it abstract:

```suggestion
  /** List of User IDs to lookup. */
```